### PR TITLE
Update gir submodule

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
It'll allow to have bitfields doc in the release.

### CODE REVIEW

**Comment by EPashkin** [comment]
CI will nag without regen

**Comment by GuillaumeGomez** [comment]
I just realized that we didn't generate correctly the name for the bitflags. So it can wait a bit. :)

**Comment by GuillaumeGomez** [comment]
Updated.

File: src/auto/versions.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Generated by gir (https://github.com/gtk-rs/gir @ 5a5b8f5)
+Generated by gir (https://github.com/gtk-rs/gir @ 3f2840b+)
```
Comment (EPashkin) [inline src/auto/versions.txt:1]: `+` not needed

**Pull request merged**